src/components/appBar/AppBar.tsx

```diff
@@ -27,11 +31,29 @@ const AppBar = ({ title, showBackButton = false, topPanelHeight }: Props) => {
   const { classes } = useStyles();
   const history = useNavigate();
   const { i18n } = useTranslation();
+  const { t } = useTranslation();
   const { appBarTitle } = useAppBarTitle();
+  const [showSnackbar] = useSnackbar();
 
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const [selectedSystem, setSelectedSystem] = useState<string>(() => sessionStorage.getItem(SELECTED_SYSTEM) || "");
+  const [systems, setSystems] = useState<System[]>([]);
+
   const isMenuOpen = Boolean(anchorEl);
 
+  React.useEffect(() => {
+    const fetchSystems = async () => {
+      systemService
+        .findAll()
+        .then((value) => setSystems(value))
+        .catch((reason) => showSnackbar(reason, SnackbarType.ERROR));
```

Comment:
I would not show such a raw error to the end user, maybe something like "Unexpected error occurred".

Reply:
Thanks, i'll change it